### PR TITLE
Optimize FixedTop scrolling

### PR DESCRIPTION
**Pull request by k911**

### CODE REVIEW

File: components/mixins/FixedTop.js [hunk 1]
```diff
@@ -8,36 +9,31 @@ export default {
...
-      this.currScrollY = window.scrollY
-
-      if (!this.isMobileMenuShown && this.isEligibleForScroll && this.isScrollingDown) {
-        this.hideFixedTop()
-      } else if (this.isScrollingUp) {
-        this.showFixedTop()
+      if (!this.isMobileMenuShown && this.isEligibleForScroll(curr)) {
+        if (this.isScrollingUp(curr)) {
+          this.showFixedTop()
+        } else {
```
Comment (FieryCod) [inline components/mixins/FixedTop.js]: It's wrong it should not scroll down when this.currScroll = this.prevScroll also i do not like the double if. Please change it back

**Comment by FieryCod** [comment]
@k911 As I said it's not a problem of fixedTop. It's a problem of v-prevent-scroll.

Here is the prove:

When job-offers-widget is rendered then the warning occurs:
image


When **job-offers-widget** is not rendered then warning does not occur:
image


**To conclude:**
There is no problem with fixedTop. There is a problem with job-offers-widget which uses **v-prevent-parent-scroll**.
Check my comment above. Your change does only this that fixedTop flickers.

Please close the **PR** and create a new one which does not change fixedTop but **v-prevent-parent-scroll**

**Comment by k911** [comment]
@FieryCod 
I know, but this one could be optimized as well. WIP = Work in progress.

**Comment by FieryCod** [comment]
#84 Please check it. If the issue is connected with passive events please update only it and provide some sort of prove that it's really helping.

**Comment by FieryCod** [comment]
Taking the work in behalf of @k911 .

**Pull request merged**